### PR TITLE
Release scripts

### PR DESCRIPTION
**Pull request by dcocchia**
This creates two new scripts for use when building and releasing a new version of Snacks.
kapture 2018-06-13 at 17 48 29

Note: video shows tests running twice. This is an odd consequence of switching to the master branch and running npm commands. Once this branch is merged, that issue should go away.

### CODE REVIEW

File: CONTRIBUTING.MD [hunk 1]
```diff
@@ -141,3 +141,10 @@ or
 ```
 npm run styleguide:build
 ```
+
+## Releasing a new version of Snacks
+1. `npm run build_release`
```
Comment (tmarshall) [inline CONTRIBUTING.MD]: why not `release:build` and `release:publish` to be consistent w/ `test:lint`?

File: scripts/release/build_release.js [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+    return false
+  }
+
+  return true
+}
+
+const confirmVersion = {
+  name: 'Is this a patch, minor or major version?',
+  type: 'string',
+  pattern: /patch|Patch|minor|Minor|Major|major|/,
```
Comment (tmarshall) [inline scripts/release/build_release.js]: looks like an extra `|` was added at the end

Comment (tmarshall) [reply scripts/release/build_release.js]: also may be worth adding `^` and `$`

something like `/^\s*(?:[pP]atch|[mM]inor|[mM]ajor)\s*$/`

Comment (tmarshall) [reply scripts/release/build_release.js]: or just `/^\s*(?:major|minor|patch)\s*$/i`

File: CONTRIBUTING.MD (hunk 1 above)
Comment (dcocchia) [reply CONTRIBUTING.MD]: Good point 👍

**Comment by apiv** [comment]
Nice!

**Pull request merged**